src/components/network-modification-table/renderers/root-network-chip-cell.tsx

```
@@ -0,0 +1,143 @@
+    }, [modificationUuid, rootNetwork.rootNetworkUuid, setModificationsToExclude, updateStatus]);
+
+    return (
+        <ActivableChip
+            label={rootNetwork.tag}
+            tooltipMessage={rootNetwork.name}
+            isActivated={isModificationActivated}
+            isDisabled={isLoading || isDisabled}
+            onClick={handleModificationActivationByRootNetwork}
+        />
+    );
+};
```

ESLint

[error] Function component is not a function declaration

(react/function-component-definition)